www/conf/sites-available/20-static-and-api.conf.var

```diff
@@ -22,7 +22,7 @@ server {
 		limit_req zone=perip-api burst=10 nodelay;
 		expires epoch;
 		etag off;
-		add_header Strict-Transport-Security "max-age=31536000; includeSubdomains";
+		add_header Strict-Transport-Security "max-age=31536000";
```

Comment:
I think this one can be removed - maybe an HSTS test in the e2e tests will prove useful.